File: Scores/Models/StudentScore.cs
using System.Text.Json.Serialization;

namespace Scores.Models
{
    public class StudentScore
    {
        [JsonPropertyName("learning_objective")]
        public string LearningObjective { get; set; }

        [JsonPropertyName("score")]
        public string Score { get; set; }
    }
}

File: scores-php/src/Models/StudentScore.php
<?php

namespace Scores\Models;

class StudentScore
{
    public function __construct(
        public string $learningObjective = '',
        public string $score = ''
    ) {}
    
    public function toArray(): array
    {
        return [
            'learning_objective' => $this->learningObjective,
            'score' => $this->score
        ];
    }
}

